include/RAJA/internal/get_platform.hpp

```diff
@@ -2,7 +2,7 @@
 #define RAJA_get_platform_HPP
 
 #include "RAJA/util/Operators.hpp"
-#include "RAJA/internal/LegacyCompatibility.hpp"
+#include "RAJA/internal/foldl.hpp"
```

Comment:
Is the LegacyCompatibility.hpp file gone now? If so, I like it.

Reply:
yes!  the only thing that we needed, that wasn't in camp, was foldl (but Tom discussed putting that in camp... perhaps I'll do a PR for that)